manage/update_manifest.py

```diff
@@ -0,0 +1,18 @@
+"""Update the manifest file."""
+import sys
+import json
+import os
+def update_manifest():
+    """Update the manifest file."""
+    version = "0.0.0"
+    for index, value in enumerate(sys.argv):
+        if value in ["--version", "-V"]:
+            version = sys.argv[index + 1]
+    with open(f"{os.getcwd()}/custom_components/simpleicons/manifest.json") as manifestfile:
+        manifest = json.load(manifestfile)
+    manifest["version"] = version
+    with open(
+        f"{os.getcwd()}/custom_components/simpleicons/manifest.json", "w"
+    ) as manifestfile:
+        manifestfile.write(json.dumps(manifest, indent=4, sort_keys=True))
+update_manifest()
```

Comment:
```suggestion
"""Update the manifest file."""

import sys
import json
import os

def update_manifest():
    """Update the manifest file."""
    
    version = "0.0.0"
    
    for index, value in enumerate(sys.argv):
        if value in ["--version", "-V"]:
            version = sys.argv[index + 1]
            
    with open(f"{os.getcwd()}/custom_components/simpleicons/manifest.json") as manifestfile:
        manifest = json.load(manifestfile)
    manifest["version"] = version
    
    with open(
        f"{os.getcwd()}/custom_components/simpleicons/manifest.json", "w"
    ) as manifestfile:
        manifestfile.write(json.dumps(manifest, indent=4, sort_keys=True))

update_manifest()
```

Reply:
This is not my code, rather a partial revert of 0904e22. I don't disagree with the suggested change, but my goal is simply to restore functionality in HACS. I'll let @vigonotion decide on any optimizations.

Reply:
My apologies, I didn't review the change very well, and it appears my copy/paste into VS Code didn't preserve formatting for some reason. I've cleaned it up. 👍